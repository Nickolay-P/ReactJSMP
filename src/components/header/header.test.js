

import React from 'react';
import {shallow} from 'enzyme'
import {Header} from './Header.js';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });


it('Header rendered correctly', () => {
  const AppComponent = shallow(
    <Header />
  );
  expect(AppComponent).toMatchSnapshot();
});