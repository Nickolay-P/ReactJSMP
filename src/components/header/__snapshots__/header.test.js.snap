// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header rendered correctly 1`] = `ShallowWrapper {}`;
